hoot-core/src/main/cpp/hoot/core/criterion/UselessElementCriterion.cpp

```diff
@@ -43,37 +43,51 @@ HOOT_FACTORY_REGISTER(ElementCriterion, UselessElementCriterion)
 bool UselessElementCriterion::isSatisfied(const boost::shared_ptr<const Element> &e) const
```

Comment:
This logging was actually for something I was helping Matt with trying to debug RemoveElementOp...so separate from #2383.